### PR TITLE
Release 2.12.0

### PR DESCRIPTION
**Pull request by attfarhan**

### CODE REVIEW

File: projects/pricing-packaging-changes.md [hunk 1]
```diff
@@ -1,17 +1,15 @@
...
 ]
 slug: pricing-and-package-changes
 heroImage: //images.ctfassets.net/le3mxztn6yoo/t4Qpcq5kA0AYM24Ws4mOk/4edf5502a936bbec90c262fa00355aed/sourcegraph-mark.png
-published: false
+published: true
 ---
 
-# Pricing and package changes
-
 Today we are excited to announce that most of Sourcegraph will be [open sourced](/blog/sourcegraph-is-now-open-source). As a part of this, more of Sourcegraph’s advanced features will be available to you for free, and one feature needed by larger companies is becoming paid. These changes will make it easier for teams to get started and get the full power of Sourcegraph right away.
```
Comment (dadlerj) [inline projects/pricing-packaging-changes.md]: @KattMingMing this text should be updated — we are no longer open sourcing today!

File: blogposts/sourcegraph-2-12-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+- [Product subscriptions, self-service checkout, and easy upgrades to Enterprise](/blog/product-subscriptions-and-license-keys)
+
+- [New product packaging and pricing](/blog/pricing-and-package-changes)
+
+- [Full Changelog (from 2.10 to 2.11)](#full-changelog-from-211-to-212)
+
+Ready to install or upgrade? [Get Sourcegraph for yourself (to get started)](/) or [deploy Sourcegraph for your team](/docs).
+
+## Full Changelog (from 2.11 to 2.12)
```
Comment (dadlerj) [inline blogposts/sourcegraph-2-12-release-notes.md]: @attfarhan I assume you synced this with the latest? My version was from last week.

Comment (attfarhan) [reply blogposts/sourcegraph-2-12-release-notes.md]: Yep

File: blogposts/sourcegraph-2-12-release-notes.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+- [Sourcegraph for GitLab](/blog/sourcegraph-for-gitlab)
+
+- [Discuss code and docs in repositories](/blog/discuss-code-and-docs-in-repositories)
+
+- [Product subscriptions, self-service checkout, and easy upgrades to Enterprise](/blog/product-subscriptions-and-license-keys)
+
+- [New product packaging and pricing](/blog/pricing-and-package-changes)
+
+- [Full Changelog (from 2.10 to 2.11)](#full-changelog-from-211-to-212)
```
Comment (ryan-blunden) [inline blogposts/sourcegraph-2-12-release-notes.md]: Should be:

[Full Changelog (from 2.11 to 2.12)]

File: blogposts/sourcegraph-2-12-release-notes.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+Ready to install or upgrade? [Get Sourcegraph for yourself (to get started)](/) or [deploy Sourcegraph for your team](/docs).
+
+## Full Changelog (from 2.11 to 2.12)
+
+### Changed
+
+- Reduced the size of in-memory data structures used for storing search results. This shdould reduce the backend memory usage of large result sets.
```
Comment (ryan-blunden) [inline blogposts/sourcegraph-2-12-release-notes.md]: Typo: "shdould"

**Pull request merged**